fritzconnection/tests/test_soaper.py

```diff
@@ -79,9 +79,23 @@ def test_raise_fritzconnection_error(error_code, exception):
     """check for exception raising depending on the error_code"""
     content = content_template.format(error_code=error_code)
     response = Response()
+    response.status_code = 500
     response.content = content.encode()
     pytest.raises(exception, raise_fritzconnection_error, response)
 
+def test_raise_fritzauthorization_error():
+    """check for exception raising depending on the error_code"""
```

Comment:
Needs better doc-string, because the raised exception depend on the `status-code` if the content is `html`. If the content is `xml` the exception depends on the `errorCode` transported by the content. (Yes, it's a weird protocol that gets used here.)

Reply:
changed the doc string to `check for exception raising depending on the html status code.`